peerblock.json

```diff
@@ -48,13 +48,13 @@
         "   #   -If no newer release than 1.1_r691, arbitrarily update PeerBlock to 1.1_r691",
         "   #    from github repo retia-adolf/dormant/, cause its official link is dead",
         "if ($version -lt 691) {",
-        "   $downloadJsonDir = $(basedir $global)",
+        "   cd \"$dir\"",
         "   if ($global) {",
         "       $globalArg = '--global'",
         "   }",
-        "   wget --quiet https://github.com/retia-adolf/dormant/raw/master/peerblock.json $downloadJsonDir",
-        "   $powershell = [System.Diagnotics.Process]::GetCurrentProcess().Path",
-        "   Start-Process $powershell -ArgumentList \"-Command\", \"scoop uninstall peerblock $globalArg `; if ($?) { scoop install $downloadJsonDir\\peerblock.json $globalArg }\" -WindowStyle Hidden",
+        "   wget https://github.com/retia-adolf/dormant/raw/master/peerblock.json",
```

Comment:
We could just add version 691 instead of cramming a self-update inside `post_install`.
Also, the URL in `https://github.com/retia-adolf/dormant/raw/master/peerblock.json` to `peerblock-setup_v1.1+_r691.exe` ends in a 404.

Reply:
Add it to our binary-mirror: 
https://github.com/scoopinstaller/binary-mirror/commit/8a13f763d2f03338491284adf90d055a754daa8e

Reply:
Thanks! Well, but don't know which bucket should version 691's json put in. scoop-version seems to be for old version apps.

Reply:
If it's the last official version that put it this one.